frontend/src/main/components/PersonalSchedules/PersonalScheduleEvent.js

```diff
@@ -0,0 +1,123 @@
+import React, { useEffect, useState } from "react";
+import {
+  ButtonGroup,
+  Card,
+  OverlayTrigger,
+  Popover,
+  Button,
+} from "react-bootstrap";
+
+export default function SchedulerEvents({ event, eventColor, borderColor }) {
+  const [style, setStyle] = useState({});
+
+  const testId = "SchedulerEvent";
+
+  const convertTimeToMinutes = (time) => {
+    const [timePart, modifier] = [time.slice(0, -2), time.slice(-2)];
+    let [hours, minutes] = timePart.split(":").map(Number);
+
+    if (modifier === "PM" && hours !== 12) {
+      hours += 12;
+    } else if (modifier === "AM" && hours === 12) {
+      hours = 0;
+    }
+
+    return hours * 60 + minutes;
+  };
+
+  // Stryker disable all : hard to test for specfic styles
+  useEffect(() => {
+    const startMinutes = convertTimeToMinutes(event.startTime);
+    const endMinutes = convertTimeToMinutes(event.endTime);
+    const height = endMinutes - startMinutes;
+    const topPosition = startMinutes + 94;
+
+    setStyle({
+      event: {
+        position: "absolute",
+        top: `${topPosition}px`,
+        height: `${height}px`,
+        width: "100%",
+        backgroundColor: eventColor,
+        border: `2px solid ${borderColor}`,
+        zIndex: 1,
+        padding: "2px",
+        justifyContent: "center",
+        alignItems: "left",
+      },
```

Comment:
I'd prefer to see more of this factored out into CSS classes and defined in this file:

https://github.com/ucsb-cs156-s25/proj-courses-s25-02/blob/main/frontend/src/index.css

That will avoid having to have so much code under a stryker exception.

Some of this is set programmatically, but a lot of it is hard coded.  The hard coded stuff can and should all be moved into CSS classes defined in index.css.

If you need help navigating how to do this, ask for it on the proj-courses slack channel, and we'll try to help as best we can.